recipe/meta.yaml

```diff
@@ -35,7 +35,7 @@ requirements:
     - clang_osx-64  # [osx]
     - m2w64-toolchain  # [win]
     - matplotlib >=2.0.2
-    - netcdf4 >=1.1.9
+    - netcdf4 >=1.1.9,<=1.4.1
```

Comment:
If parcels works with the latest version this should not be pinned due to performance reasons. The user can also specify another version in their working environment.

Reply:
The performance is so badly impeded that we can consider Parcels broken with the latest release of  netCDF4 and on Python 3.  (I do not yet understand why this is happening. But a factor of 100 in execution times quickly gets you from "feasible" to "never going to finish".)

I'd go for this: Pin to <1.4.2 for Python 3 only and the remove the restriction as soon as we know what's going on?

Reply:
Thanks btw for having a look here!

Reply:
> The performance is so badly impeded that we can consider Parcels broken with the latest release of netCDF4 and on Python 3.

While I disagree with that I'm OK with the changes b/c I don't really have a stake on this, but not that performance is not a reason to declare brokenness. People may be more frustrate for not being able to install the latest version. Also, if the performance issue is resolved, it will take a while for updates to parcels to happen b/c you will need to do another build to unpin.

Note that you can always specify the faster netcdf4 in your env though, but with this PR a user that wants the latest won't have that option.

> (I do not yet understand why this is happening. But a factor of 100 in execution times quickly gets you from "feasible" to "never going to finish".)

Probably the default masked array which I believe will be revert/fixed soon. I would open an issue upstream to report that.

> I'd go for this: Pin to <1.4.2 for Python 3 only and the remove the restriction as soon as we know what's going on?

It is up to the maintainers. IMHO you should not pin.

PS: it would worse to pin only for Python 3 b/c it would be quite confusing if people need to inspect their envs.

Reply:
Having thought about this a bit more, I think I understand and fully support your POV on this PR.  So I'll close this without merging.